### PR TITLE
ISSD-1580 [Marketplace] Fix layout divergencies

### PR DESCRIPTION
**Pull request by claricebrasil**

### CODE REVIEW

**Comment by claricebrasil** [comment]
ci:test:sf

**Comment by silasbsi** [comment]
We just need to pay attention to the `padding: 64px 423px;`. We'll probably need to revisit this in the future. @kevenleone  are you refactoring this?

**Comment by kevenleone** [comment]
@silasbsi I asked in person to @claricebrasil to refactor this content and remove the crazy padding, she'll update the PR with a better solution

thanks for reviewing 🎉

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/DashboardTable/DashboardEmptyTable.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@
 	border-radius: 10px;
 	display: flex;
 	flex-direction: column;
-	padding: 64px 468px;
+	padding: 64px 423px;
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/DashboardTable/DashboardEmptyTable.scss:7]: @claricebrasil can you find a better solutions for this?

```suggestion
	
```

**Review by kevenleone — CHANGES_REQUESTED** [review]

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/DashboardTable/DashboardEmptyTable.scss (hunk 1 above)
Comment (claricebrasil) [reply workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/DashboardTable/DashboardEmptyTable.scss]: Sure! I made the changes to fix the padding issue. Thank you!

**Comment by claricebrasil** [comment]
> We just need to pay attention to the `padding: 64px 423px;`. We'll probably need to revisit this in the future. @kevenleone are you refactoring this?

Done! Thank you, Silas!

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Comment by claricebrasil** [comment]
resent: https://github.com/brianchandotcom/liferay-portal/pull/143499

**Pull request closed without merging**